src/base-commands/base-command.js

```diff
@@ -64,7 +64,14 @@ class BaseCommand extends Command {
       // User/API errors
       this.logger.error(error.message);
       this.logger.debug(error.stack);
+
+      if (this.flags['cli-output-format'] === 'json') {
```

Comment:
Why only for json output?

Reply:
It was recommended to do it this way temporarily in the ticket description. I believe it has something to do with how outputProcessor works, but I'm not entirely sure. This would be temporary until `this.ouput(error.details)` accounts for formatting.